### ISSUE TITLE
Add support for HTTP request headers

### ISSUE THREAD

**Issue by flashback19** [michael-russin/webview2-control#4]

**Comment by michael-russin** [comment]
Do you mean support headers in the initial navigation?

**Comment by flashback19** [comment]
Yes, i need to send authenticaton token in HTTP request header (sign in).

I was able to access and change the requestheaders at NavigationStarted event, but i need to change it before the request was sent to the server.

**Comment by michael-russin** [comment]
I have a similar request open with the webview2 team (https://github.com/MicrosoftEdge/WebViewFeedback/issues/69) Once they support it I can add it to this project.

**Comment by cenright** [comment]
Not sure if this is already well known, but I needed to hack up headers and content also and was sad to see NavigationStarted provides no joy.

If you are desperate, you can modify headers / content:
Add filter for web resources using AddWebResourceRequestedFilter
and subscribing to event WebResourceRequested 
-> modifying headers / content there for your desired Uri

Not sure if I failed, but I could NOT make this work for microsoft's winform control implementation.   So bravo michael-russin
